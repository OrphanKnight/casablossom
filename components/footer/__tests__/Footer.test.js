import { render } from '@testing-library/react';
import Footer from '../index';

it('renders Footer component', () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
})