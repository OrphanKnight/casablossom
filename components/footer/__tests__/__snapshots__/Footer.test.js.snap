// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="footer__container"
    >
      <div
        class="footer__links"
      >
        <ul>
          <a
            href="#"
          >
            <svg
              fill="none"
              height="46"
              id="logo-25"
              viewBox="0 0 147 46"
              width="147"
              xmlns="http://www.w3.org/2000/svg"
            >
               
              <path
                class="ccompli1"
                clip-rule="evenodd"
                d="M72.6756 7.40747C74.0634 6.01977 76.3134 6.01977 77.7012 7.40747C79.089 8.79527 79.089 11.0453 77.7012 12.4331L69.2403 20.894C66.8111 17.6384 67.075 13.0082 70.032 10.0512L72.6756 7.40747Z"
                fill="#4F9E91"
                fill-rule="evenodd"
              />
               
              <path
                class="ccustom"
                clip-rule="evenodd"
                d="M84.771 26.4967C86.159 27.8844 86.159 30.1345 84.771 31.5222C83.384 32.91 81.1336 32.91 79.7458 31.5222L71.2849 23.0614C74.5405 20.6322 79.1707 20.896 82.1277 23.853L84.771 26.4967Z"
                fill="#EBE1C5"
                fill-rule="evenodd"
              />
               
              <path
                class="ccompli2"
                clip-rule="evenodd"
                d="M65.6821 38.5924C64.2944 39.9802 62.0443 39.9802 60.6566 38.5924C59.2688 37.2046 59.2688 34.9546 60.6566 33.5668L69.1174 25.106C71.5466 28.3616 71.2828 32.9918 68.3258 35.9487L65.6821 38.5924Z"
                fill="#EBCC6E"
                fill-rule="evenodd"
              />
               
              <path
                class="ccustom"
                clip-rule="evenodd"
                d="M53.5865 19.5032C52.1987 18.1154 52.1987 15.8654 53.5865 14.4776C54.9743 13.0898 57.2243 13.0898 58.6121 14.4776L67.073 22.9385C63.8174 25.3677 59.1871 25.1038 56.2302 22.1469L53.5865 19.5032Z"
                fill="#ED834E"
                fill-rule="evenodd"
              />
               
              <path
                class="ccustom"
                clip-rule="evenodd"
                d="M70.7849 37.5562C72.8312 35.281 73.7823 32.3697 73.638 29.5033L77.7014 33.5667C77.7115 33.5768 77.7217 33.5869 77.7319 33.597V36.498C77.7319 38.4606 76.1409 40.0516 74.1783 40.0516C72.5842 40.0516 71.2353 39.002 70.7849 37.5562Z"
                fill="#ED834E"
                fill-rule="evenodd"
              />
               
              <path
                class="ccompli1"
                clip-rule="evenodd"
                d="M54.6226 24.6058C56.8977 26.6522 59.809 27.6033 62.6754 27.459L58.6121 31.5224C58.6019 31.5325 58.5918 31.5427 58.5818 31.5528H55.6808C53.7182 31.5529 52.1272 29.9618 52.1272 27.9992C52.1272 26.4052 53.1768 25.0562 54.6226 24.6058Z"
                fill="#4F9E91"
                fill-rule="evenodd"
              />
               
              <path
                class="ccompli2"
                clip-rule="evenodd"
                d="M83.735 21.3942C81.4601 19.3478 78.5488 18.3967 75.6824 18.5409L79.7457 14.4776C79.7559 14.4675 79.7659 14.4573 79.776 14.4471H82.6771C84.64 14.4471 86.231 16.0382 86.231 18.0008C86.231 19.5949 85.181 20.9438 83.735 21.3942Z"
                fill="#EBCC6E"
                fill-rule="evenodd"
              />
               
              <path
                class="ccustom"
                clip-rule="evenodd"
                d="M67.573 8.44368C65.5266 10.7188 64.5756 13.6301 64.7198 16.4966L60.6565 12.4332C60.6463 12.4231 60.6362 12.413 60.626 12.4029V9.50188C60.626 7.53928 62.217 5.94818 64.1796 5.94818C65.7737 5.94818 67.1226 6.99788 67.573 8.44368Z"
                fill="#EBE1C5"
                fill-rule="evenodd"
              />
               
              <path
                class="cneutral"
                clip-rule="evenodd"
                d="M64.1798 3.05701C60.6204 3.05701 57.7349 5.94251 57.7349 9.50191V10.7554C55.5935 10.1959 53.2203 10.7551 51.5422 12.4332C49.0253 14.9501 49.0253 19.0308 51.5422 21.5477L52.4285 22.434C50.5187 23.5526 49.2361 25.6261 49.2361 27.9992C49.2361 31.5586 52.1216 34.4441 55.681 34.4441H56.9344C56.3749 36.5855 56.9341 38.9587 58.6122 40.6367C61.1291 43.1536 65.2098 43.1536 67.7267 40.6367L70.3703 37.9931C72.7024 35.661 73.7916 32.5566 73.6379 29.5032L77.7014 33.5666C80.2183 36.0835 84.2991 36.0835 86.8161 33.5666C89.3331 31.0497 89.3331 26.969 86.8161 24.4522L85.9301 23.566C87.8391 22.4474 89.1221 20.3739 89.1221 18.0008C89.1221 14.4414 86.2361 11.5559 82.6771 11.5559H81.4236C81.9832 9.41441 81.4239 7.04121 79.7458 5.36311C77.229 2.84621 73.1483 2.84621 70.6314 5.36311L67.9877 8.00681C65.6556 10.3389 64.5664 13.4432 64.7201 16.4966L60.6567 12.4332C60.6465 12.4231 60.6364 12.413 60.6262 12.4029V9.50191C60.6262 7.53931 62.2172 5.94831 64.1798 5.94831C65.7739 5.94831 67.1228 6.99791 67.5732 8.44371C67.7067 8.29531 67.8448 8.14961 67.9876 8.00681L69.745 6.24941C68.6264 4.33961 66.5529 3.05701 64.1798 3.05701ZM79.7761 14.4472C79.766 14.4573 79.756 14.4675 79.7458 14.4776L75.6825 18.5409C78.549 18.3967 81.4602 19.3477 83.7351 21.3942C85.1811 20.9438 86.2311 19.5948 86.2311 18.0008C86.2311 16.0382 84.6401 14.4472 82.6771 14.4472H79.7761ZM58.5818 31.5528C58.5919 31.5426 58.602 31.5324 58.6122 31.5223L62.6755 27.4589C59.8091 27.6032 56.8979 26.6521 54.6228 24.6058C53.177 25.0562 52.1273 26.4051 52.1273 27.9992C52.1273 29.9618 53.7184 31.5528 55.681 31.5528H58.5818ZM54.512 24.5048C54.4017 24.4029 54.293 24.2984 54.1859 24.1913V24.1914C54.293 24.2985 54.4017 24.4029 54.512 24.5048ZM74.1783 42.943C71.8051 42.943 69.7316 41.6603 68.613 39.7505L70.3703 37.9931C70.5132 37.8503 70.6513 37.7046 70.7848 37.5562C71.2352 39.002 72.5841 40.0517 74.1783 40.0517C76.1409 40.0517 77.7319 38.4607 77.7319 36.4981V33.5971C78.5647 34.419 79.5657 34.9682 80.6232 35.2445V36.4981C80.6232 40.0575 77.7377 42.943 74.1783 42.943ZM77.7014 7.40751C76.3136 6.01981 74.0636 6.01981 72.6758 7.40751L70.0321 10.0512C67.0752 13.0082 66.8113 17.6384 69.2405 20.894L77.7014 12.4331C79.0892 11.0453 79.0892 8.79531 77.7014 7.40751ZM84.7711 31.5222C86.1591 30.1344 86.1591 27.8844 84.7711 26.4966L82.1277 23.8529C79.1708 20.896 74.5405 20.6321 71.2849 23.0613L79.7458 31.5222C81.1336 32.91 83.3841 32.91 84.7711 31.5222ZM65.6822 38.5923C64.2944 39.9801 62.0444 39.9801 60.6566 38.5923C59.2689 37.2045 59.2689 34.9545 60.6566 33.5667L69.1175 25.1058C71.5467 28.3614 71.2828 32.9917 68.3259 35.9486L65.6822 38.5923ZM53.5866 14.4776C52.1989 15.8654 52.1989 18.1154 53.5866 19.5032L56.2303 22.1469C59.1873 25.1038 63.8175 25.3677 67.0731 22.9385L58.6122 14.4776C57.2244 13.0899 54.9744 13.0899 53.5866 14.4776Z"
                fill="#3D3A2C"
                fill-rule="evenodd"
              />
               
              <path
                class="cneutral"
                d="M1.53271 16.0898H4.69331V25.911H10.1217V28.5716H1.53271V16.0898Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M15.8603 28.8395C14.8008 28.8395 13.8722 28.649 13.0746 28.2681C12.277 27.8752 11.658 27.3395 11.2175 26.661C10.789 25.9705 10.5747 25.1789 10.5747 24.286C10.5747 23.3932 10.789 22.6075 11.2175 21.929C11.658 21.2385 12.277 20.7028 13.0746 20.3219C13.8722 19.929 14.8008 19.7326 15.8603 19.7326C16.9198 19.7326 17.8483 19.929 18.6459 20.3219C19.4435 20.7028 20.0566 21.2385 20.4851 21.929C20.9256 22.6075 21.1458 23.3932 21.1458 24.286C21.1458 25.1789 20.9256 25.9705 20.4851 26.661C20.0566 27.3395 19.4435 27.8752 18.6459 28.2681C17.8483 28.649 16.9198 28.8395 15.8603 28.8395ZM15.8603 26.3931C16.4912 26.3931 16.9912 26.2086 17.3602 25.8396C17.7412 25.4586 17.9316 24.9408 17.9316 24.286C17.9316 23.6313 17.7412 23.1194 17.3602 22.7504C16.9912 22.3694 16.4912 22.179 15.8603 22.179C15.2293 22.179 14.7234 22.3694 14.3425 22.7504C13.9734 23.1194 13.7889 23.6313 13.7889 24.286C13.7889 24.9408 13.9734 25.4586 14.3425 25.8396C14.7234 26.2086 15.2293 26.3931 15.8603 26.3931Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M27.0492 32.6787C26.4778 32.6787 25.9004 32.6251 25.3171 32.518C24.7457 32.4108 24.2516 32.268 23.835 32.0894V29.9109C24.7754 30.3156 25.8468 30.518 27.0492 30.518C27.9182 30.518 28.5432 30.3394 28.9241 29.9823C29.317 29.6252 29.5134 29.1014 29.5134 28.4109V27.8931C29.1682 28.1907 28.7396 28.4228 28.2277 28.5895C27.7277 28.7562 27.192 28.8395 26.6206 28.8395C25.7159 28.8395 24.9183 28.649 24.2278 28.2681C23.5374 27.8871 22.9957 27.3514 22.6029 26.661C22.2219 25.9705 22.0315 25.1789 22.0315 24.286C22.0315 23.3813 22.2219 22.5837 22.6029 21.8933C22.9838 21.2028 23.5195 20.673 24.21 20.304C24.9004 19.9231 25.6921 19.7326 26.5849 19.7326C27.8111 19.7326 28.8229 20.1076 29.6205 20.8576V20.0004H32.6562V27.9824C32.6562 29.4704 32.1978 30.6252 31.2812 31.4466C30.3765 32.268 28.9658 32.6787 27.0492 32.6787ZM27.4063 26.411C27.8944 26.411 28.3289 26.286 28.7098 26.036C29.0908 25.786 29.3586 25.4527 29.5134 25.036V23.5361C29.3467 23.1194 29.0729 22.792 28.692 22.5539C28.311 22.304 27.8825 22.179 27.4063 22.179C26.7635 22.179 26.2456 22.3694 25.8528 22.7504C25.4599 23.1194 25.2635 23.6253 25.2635 24.2682C25.2635 24.9229 25.4599 25.4467 25.8528 25.8396C26.2456 26.2205 26.7635 26.411 27.4063 26.411Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M38.739 28.8395C37.6795 28.8395 36.751 28.649 35.9534 28.2681C35.1558 27.8752 34.5367 27.3395 34.0963 26.661C33.6677 25.9705 33.4534 25.1789 33.4534 24.286C33.4534 23.3932 33.6677 22.6075 34.0963 21.929C34.5367 21.2385 35.1558 20.7028 35.9534 20.3219C36.751 19.929 37.6795 19.7326 38.739 19.7326C39.7985 19.7326 40.727 19.929 41.5246 20.3219C42.3222 20.7028 42.9353 21.2385 43.3639 21.929C43.8043 22.6075 44.0246 23.3932 44.0246 24.286C44.0246 25.1789 43.8043 25.9705 43.3639 26.661C42.9353 27.3395 42.3222 27.8752 41.5246 28.2681C40.727 28.649 39.7985 28.8395 38.739 28.8395ZM38.739 26.3931C39.3699 26.3931 39.8699 26.2086 40.239 25.8396C40.6199 25.4586 40.8104 24.9408 40.8104 24.286C40.8104 23.6313 40.6199 23.1194 40.239 22.7504C39.8699 22.3694 39.3699 22.179 38.739 22.179C38.1081 22.179 37.6021 22.3694 37.2212 22.7504C36.8521 23.1194 36.6676 23.6313 36.6676 24.286C36.6676 24.9408 36.8521 25.4586 37.2212 25.8396C37.6021 26.2086 38.1081 26.3931 38.739 26.3931Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M94.4319 16.0898H97.5929V28.5716H94.4319V16.0898Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M99.1931 20.0004H102.229V20.8576C103.026 20.1076 104.038 19.7326 105.264 19.7326C106.157 19.7326 106.949 19.9231 107.639 20.304C108.33 20.673 108.865 21.2028 109.246 21.8933C109.627 22.5718 109.818 23.3635 109.818 24.2682C109.818 25.161 109.627 25.9586 109.246 26.661C108.865 27.3514 108.33 27.8871 107.639 28.2681C106.961 28.649 106.169 28.8395 105.264 28.8395C104.693 28.8395 104.145 28.7502 103.622 28.5716C103.11 28.405 102.681 28.1728 102.336 27.8752V32.4822H99.1931V20.0004ZM104.443 26.3931C105.086 26.3931 105.604 26.2027 105.996 25.8217C106.389 25.4408 106.586 24.9229 106.586 24.2682C106.586 23.6134 106.389 23.1016 105.996 22.7325C105.604 22.3516 105.086 22.1611 104.443 22.1611C103.967 22.1611 103.538 22.2861 103.157 22.5361C102.776 22.7742 102.503 23.1016 102.336 23.5182V25.036C102.503 25.4527 102.776 25.786 103.157 26.036C103.538 26.2741 103.967 26.3931 104.443 26.3931Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M114.503 28.8395C113.813 28.8395 113.093 28.7681 112.343 28.6252C111.593 28.4943 110.997 28.3038 110.557 28.0538V25.7503C111.045 26.036 111.628 26.2741 112.307 26.4646C112.997 26.6431 113.628 26.7324 114.2 26.7324C114.616 26.7324 114.92 26.7026 115.11 26.6431C115.301 26.5836 115.396 26.4646 115.396 26.286C115.396 26.1431 115.331 26.03 115.2 25.9467C115.081 25.8515 114.854 25.7562 114.521 25.661C114.2 25.5539 113.825 25.4408 113.396 25.3217C112.67 25.1074 112.099 24.8872 111.682 24.661C111.265 24.4348 110.956 24.1491 110.753 23.8039C110.551 23.4587 110.45 23.0242 110.45 22.5004C110.45 21.5718 110.813 20.8814 111.539 20.429C112.277 19.9647 113.283 19.7326 114.557 19.7326C115.176 19.7326 115.825 19.7981 116.503 19.929C117.194 20.06 117.711 20.2326 118.057 20.4469V22.7147C117.7 22.4766 117.223 22.2801 116.628 22.1254C116.045 21.9706 115.497 21.8933 114.985 21.8933C114.581 21.8933 114.265 21.9349 114.039 22.0182C113.813 22.0897 113.7 22.2206 113.7 22.4111C113.7 22.5897 113.813 22.7266 114.039 22.8218C114.265 22.917 114.67 23.0361 115.253 23.1789C115.634 23.2742 115.795 23.3158 115.735 23.3039C116.497 23.5063 117.086 23.7325 117.503 23.9825C117.92 24.2325 118.211 24.5301 118.378 24.8753C118.557 25.2086 118.646 25.6372 118.646 26.161C118.646 26.9824 118.289 27.6371 117.575 28.1252C116.872 28.6014 115.848 28.8395 114.503 28.8395Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M123.338 28.8395C122.16 28.8395 121.273 28.5121 120.678 27.8574C120.095 27.2026 119.803 26.3396 119.803 25.2682V20.0004H122.946V24.911C122.946 25.8991 123.392 26.3931 124.285 26.3931C124.69 26.3931 125.059 26.2979 125.392 26.1074C125.725 25.9169 125.999 25.6372 126.213 25.2682V20.0004H129.356V28.5716H126.321V27.6431C125.952 28.0359 125.505 28.3335 124.981 28.5359C124.458 28.7383 123.91 28.8395 123.338 28.8395Z"
                fill="#3D3A2C"
              />
               
              <path
                class="cneutral"
                d="M130.965 20.0004H133.876V20.929C134.233 20.5481 134.644 20.2564 135.108 20.054C135.572 19.8397 136.066 19.7326 136.59 19.7326C137.233 19.7326 137.745 19.8278 138.126 20.0183C138.519 20.2088 138.834 20.4945 139.072 20.8754C139.429 20.5183 139.864 20.2385 140.376 20.0362C140.888 19.8338 141.406 19.7326 141.929 19.7326C143.06 19.7326 143.864 20.0243 144.34 20.6076C144.816 21.179 145.054 22.0063 145.054 23.0896V28.5716H142.054V23.5361C142.054 23.0599 141.965 22.7147 141.786 22.5004C141.608 22.2861 141.31 22.179 140.894 22.179C140.358 22.179 139.923 22.3694 139.59 22.7504C139.602 22.9527 139.608 23.2623 139.608 23.6789V28.5716H136.608V23.4111C136.608 22.9706 136.531 22.6551 136.376 22.4647C136.221 22.2742 135.947 22.179 135.555 22.179C134.947 22.179 134.424 22.5182 133.983 23.1968V28.5716H130.965V20.0004Z"
                fill="#3D3A2C"
              />
               
            </svg>
          </a>
          <li>
            <a
              href=""
            >
              Flowers always make people better, happier, they are sunshine and medicine for the soul.
            </a>
          </li>
          <li>
            <a
              href=""
            >
              . 
            </a>
          </li>
          <li>
            <a
              href=""
            >
              Calle Ocho, Miami, Florida
            </a>
          </li>
          <li>
            <a
              href=""
            >
              +178843222222
            </a>
          </li>
        </ul>
        <ul>
          <b>
            Working hours
          </b>
          <li>
            <a
              href=""
            >
              Monday - Friday: 
            </a>
          </li>
          <li>
            <a
              href=""
            >
              9AM - 5PM
            </a>
          </li>
          <li>
            <a
              href=""
            >
              Saturday:
            </a>
          </li>
          <li>
            <a
              href=""
            >
              9Am - 3PM
            </a>
          </li>
          <li>
            <a
              href=""
            >
              Sunday: Closed
            </a>
          </li>
        </ul>
        <ul>
          <b>
            Customer service
          </b>
          <li>
            <a
              href="/services/customer"
            >
              Customer service
            </a>
          </li>
          <li>
            <a
              href="/services/returns"
            >
              Return Policy
            </a>
          </li>
          <li>
            <a
              href="/services/shipping"
            >
              Shipping Policy
            </a>
          </li>
          <li>
            <a
              href="/contact"
            >
              Events
            </a>
          </li>
        </ul>
      </div>
      <div
        class="footer__socials"
      >
        <section>
          <h3>
            STAY CONNECTED
          </h3>
          <ul>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 320 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
                  />
                </svg>
              </a>
            </li>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8 0C5.829 0 5.556.01 4.703.048 3.85.088 3.269.222 2.76.42a3.917 3.917 0 0 0-1.417.923A3.927 3.927 0 0 0 .42 2.76C.222 3.268.087 3.85.048 4.7.01 5.555 0 5.827 0 8.001c0 2.172.01 2.444.048 3.297.04.852.174 1.433.372 1.942.205.526.478.972.923 1.417.444.445.89.719 1.416.923.51.198 1.09.333 1.942.372C5.555 15.99 5.827 16 8 16s2.444-.01 3.298-.048c.851-.04 1.434-.174 1.943-.372a3.916 3.916 0 0 0 1.416-.923c.445-.445.718-.891.923-1.417.197-.509.332-1.09.372-1.942C15.99 10.445 16 10.173 16 8s-.01-2.445-.048-3.299c-.04-.851-.175-1.433-.372-1.941a3.926 3.926 0 0 0-.923-1.417A3.911 3.911 0 0 0 13.24.42c-.51-.198-1.092-.333-1.943-.372C10.443.01 10.172 0 7.998 0h.003zm-.717 1.442h.718c2.136 0 2.389.007 3.232.046.78.035 1.204.166 1.486.275.373.145.64.319.92.599.28.28.453.546.598.92.11.281.24.705.275 1.485.039.843.047 1.096.047 3.231s-.008 2.389-.047 3.232c-.035.78-.166 1.203-.275 1.485a2.47 2.47 0 0 1-.599.919c-.28.28-.546.453-.92.598-.28.11-.704.24-1.485.276-.843.038-1.096.047-3.232.047s-2.39-.009-3.233-.047c-.78-.036-1.203-.166-1.485-.276a2.478 2.478 0 0 1-.92-.598 2.48 2.48 0 0 1-.6-.92c-.109-.281-.24-.705-.275-1.485-.038-.843-.046-1.096-.046-3.233 0-2.136.008-2.388.046-3.231.036-.78.166-1.204.276-1.486.145-.373.319-.64.599-.92.28-.28.546-.453.92-.598.282-.11.705-.24 1.485-.276.738-.034 1.024-.044 2.515-.045v.002zm4.988 1.328a.96.96 0 1 0 0 1.92.96.96 0 0 0 0-1.92zm-4.27 1.122a4.109 4.109 0 1 0 0 8.217 4.109 4.109 0 0 0 0-8.217zm0 1.441a2.667 2.667 0 1 1 0 5.334 2.667 2.667 0 0 1 0-5.334z"
                  />
                </svg>
              </a>
            </li>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
                  />
                </svg>
              </a>
            </li>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8.051 1.999h.089c.822.003 4.987.033 6.11.335a2.01 2.01 0 0 1 1.415 1.42c.101.38.172.883.22 1.402l.01.104.022.26.008.104c.065.914.073 1.77.074 1.957v.075c-.001.194-.01 1.108-.082 2.06l-.008.105-.009.104c-.05.572-.124 1.14-.235 1.558a2.007 2.007 0 0 1-1.415 1.42c-1.16.312-5.569.334-6.18.335h-.142c-.309 0-1.587-.006-2.927-.052l-.17-.006-.087-.004-.171-.007-.171-.007c-1.11-.049-2.167-.128-2.654-.26a2.007 2.007 0 0 1-1.415-1.419c-.111-.417-.185-.986-.235-1.558L.09 9.82l-.008-.104A31.4 31.4 0 0 1 0 7.68v-.123c.002-.215.01-.958.064-1.778l.007-.103.003-.052.008-.104.022-.26.01-.104c.048-.519.119-1.023.22-1.402a2.007 2.007 0 0 1 1.415-1.42c.487-.13 1.544-.21 2.654-.26l.17-.007.172-.006.086-.003.171-.007A99.788 99.788 0 0 1 7.858 2h.193zM6.4 5.209v4.818l4.157-2.408L6.4 5.209z"
                  />
                </svg>
              </a>
            </li>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8 0a8 8 0 0 0-2.915 15.452c-.07-.633-.134-1.606.027-2.297.146-.625.938-3.977.938-3.977s-.239-.479-.239-1.187c0-1.113.645-1.943 1.448-1.943.682 0 1.012.512 1.012 1.127 0 .686-.437 1.712-.663 2.663-.188.796.4 1.446 1.185 1.446 1.422 0 2.515-1.5 2.515-3.664 0-1.915-1.377-3.254-3.342-3.254-2.276 0-3.612 1.707-3.612 3.471 0 .688.265 1.425.595 1.826a.24.24 0 0 1 .056.23c-.061.252-.196.796-.222.907-.035.146-.116.177-.268.107-1-.465-1.624-1.926-1.624-3.1 0-2.523 1.834-4.84 5.286-4.84 2.775 0 4.932 1.977 4.932 4.62 0 2.757-1.739 4.976-4.151 4.976-.811 0-1.573-.421-1.834-.919l-.498 1.902c-.181.695-.669 1.566-.995 2.097A8 8 0 1 0 8 0z"
                  />
                </svg>
              </a>
            </li>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M15.98 11.93c-.068-.232-.405-.395-.405-.395l-.082-.044a6.462 6.462 0 0 1-1.466-.96 5.028 5.028 0 0 1-.85-.966 3.775 3.775 0 0 1-.47-.966c-.03-.126-.027-.173 0-.238a.36.36 0 0 1 .116-.13c.187-.132.49-.33.677-.448.16-.106.3-.194.381-.252.262-.184.439-.367.544-.568a.987.987 0 0 0 .048-.827c-.143-.377-.497-.605-.946-.605-.098 0-.204.01-.306.034a4.325 4.325 0 0 0-.707.228c-.014.007-.03-.007-.03-.02.02-.507.043-1.188-.011-1.834-.051-.585-.17-1.078-.368-1.507a4.053 4.053 0 0 0-.656-.983 3.989 3.989 0 0 0-1.037-.833C9.697.207 8.88 0 7.992 0c-.888 0-1.7.207-2.419.616a4.043 4.043 0 0 0-1.037.833c-.2.231-.46.551-.656.983-.198.429-.317.922-.368 1.507-.054.65-.034 1.276-.01 1.833 0 .017-.017.028-.03.02a5.195 5.195 0 0 0-.705-.227 1.34 1.34 0 0 0-.306-.034c-.449 0-.803.228-.946.605a.975.975 0 0 0 .048.827c.105.2.286.388.544.568.082.058.218.146.381.252.18.119.476.31.664.442a.434.434 0 0 1 .129.14c.027.068.03.115-.004.248-.057.194-.19.54-.466.956a4.99 4.99 0 0 1-.85.966 6.468 6.468 0 0 1-1.456.959c-.027.014-.058.027-.092.047 0 0-.333.17-.394.388-.092.323.153.626.405.79.411.265.915.408 1.204.486.081.02.153.04.22.061a.457.457 0 0 1 .191.112c.058.072.065.164.085.266.03.17.102.38.313.527.232.16.527.17.898.187.391.014.878.034 1.432.218.259.085.49.228.762.394.565.347 1.266.78 2.466.78 1.201 0 1.909-.436 2.477-.783.268-.163.5-.306.752-.391.554-.184 1.04-.2 1.432-.218.374-.014.666-.024.898-.187.224-.156.292-.388.32-.565.016-.085.027-.163.078-.228.04-.054.139-.091.183-.108l.228-.065c.293-.078.657-.17 1.099-.422.537-.3.575-.67.517-.854z"
                  />
                </svg>
              </a>
            </li>
            <li>
              <a
                href="/"
                target="_blank"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 448 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M448,209.91a210.06,210.06,0,0,1-122.77-39.25V349.38A162.55,162.55,0,1,1,185,188.31V278.2a74.62,74.62,0,1,0,52.23,71.18V0l88,0a121.18,121.18,0,0,0,1.86,22.17h0A122.18,122.18,0,0,0,381,102.39a121.43,121.43,0,0,0,67,20.14Z"
                  />
                </svg>
              </a>
            </li>
          </ul>
        </section>
      </div>
      <div
        class="footer__newsletter"
      >
        <h3>
          SIGN UP FOR OUR NEWSLETTER
        </h3>
        <div
          class="footer__flex"
        >
          <input
            placeholder="Your Email Address"
            type="text"
          />
          <button
            class="btn_primary"
          >
            SUBSCRIBE
          </button>
        </div>
        <p>
          By clicking the SUBSCRIBE button, you are agreeing to
           
          <a
            href=""
          >
            our Privacy & Cookie Policy
          </a>
        </p>
      </div>
      <div
        class="footer__payment"
      >
        <h3>
          WE ACCEPT
        </h3>
        <div
          class="footer__flexwrap"
        >
          <img
            alt=""
            src="../../../images/payment/visa.webp"
          />
          <img
            alt=""
            src="../../../images/payment/mastercard.webp"
          />
          <img
            alt=""
            src="../../../images/payment/paypal.webp"
          />
        </div>
      </div>
      <div
        class="footer__copyright"
      >
        <section>
          ©2023 CasaBlossom All Rights Resereved.
        </section>
        <section>
          <ul>
            <li>
              <a
                href=""
              >
                Privacy Center
              </a>
            </li>
            <li>
              <a
                href=""
              >
                Privacy & Cookie Policy
              </a>
            </li>
            <li>
              <a
                href=""
              >
                Manage Cookies
              </a>
            </li>
            <li>
              <a
                href=""
              >
                Terms & Conditions
              </a>
            </li>
            <li>
              <a
                href=""
              >
                Copyright Notice
              </a>
            </li>
            <li>
              <a>
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 512 512"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M256 32C167.67 32 96 96.51 96 176c0 128 160 304 160 304s160-176 160-304c0-79.49-71.67-144-160-144zm0 224a64 64 0 1164-64 64.07 64.07 0 01-64 64z"
                  />
                </svg>
                 Miami
              </a>
            </li>
          </ul>
        </section>
      </div>
    </div>
  </footer>
</div>
`;
